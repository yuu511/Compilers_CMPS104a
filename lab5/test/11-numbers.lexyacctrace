./mk.tests: line 21: ../oc: No such file or directory
