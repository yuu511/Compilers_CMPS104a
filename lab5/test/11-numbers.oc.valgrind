==3770== Memcheck, a memory error detector
==3770== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==3770== Using Valgrind-3.13.0 and LibVEX; rerun with -h for copyright info
==3770== Command: ../oc 11-numbers.oc
==3770== 
==3770== 
==3770== HEAP SUMMARY:
==3770==     in use at exit: 0 bytes in 0 blocks
==3770==   total heap usage: 451 allocs, 451 frees, 144,086 bytes allocated
==3770== 
==3770== All heap blocks were freed -- no leaks are possible
==3770== 
==3770== For counts of detected and suppressed errors, rerun with: -v
==3770== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
