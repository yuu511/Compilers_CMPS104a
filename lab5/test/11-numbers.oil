main:      .function int 
.local int number
.local int done
return
.end
