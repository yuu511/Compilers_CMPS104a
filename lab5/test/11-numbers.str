string_set[   0]:    7296690267450279874 0x55afa9375cd8->"chr"
                   13348923631108115385 0x55afa9376698->"putstr"
string_set[   1]:   15578157668962758274 0x55afa9377a58->"done"
string_set[   5]:   12679825168387489523 0x55afa93761f8->"num"
string_set[   6]:    9317282556992397945 0x55afa93783c8->"if"
string_set[   7]:    2297668033614959926 0x55afa9377b98->"0"
string_set[  10]:    1583576014053361350 0x55afa9376858->"str"
                   13497578857207247412 0x55afa9375278->"string"
                   11597697714117577063 0x55afa9377838->"="
string_set[  13]:   12868445110721718657 0x55afa93758f8->")"
string_set[  16]:   11911796782241914524 0x55afa9373fe8->"void"
                   10159970873491820195 0x55afa93778f8->"1"
string_set[  19]:    5344662657899890615 0x55afa93759b8->";"
string_set[  20]:   15816506932648033945 0x55afa93751b8->"("
                   11868120863286193964 0x55afa9377478->"main"
string_set[  25]:   11166701219533395393 0x55afa9376e78->"getstr"
string_set[  29]:   14635384904416252381 0x55afa9378bc8->"return"
                    1370730858159036685 0x55afa93789d8->"+"
string_set[  34]:    7211130878751977441 0x55afa9375b18->"putchr"
string_set[  37]:   12291392672316406109 0x55afa9375818->"line"
                    6192799629375248351 0x55afa93782c8->"'\n'"
string_set[  39]:   18413295556186398326 0x55afa9376038->"putint"
string_set[  40]:   10319632053171904911 0x55afa9377d28->"while"
string_set[  44]:      21606623943142546 0x55afa9377638->"{"
string_set[  45]:    6142509188972423790 0x55afa93753f8->""
string_set[  47]:   13686721134187162920 0x55afa9378b38->"}"
string_set[  57]:    9386170080689500517 0x55afa9375338->"expr"
string_set[  58]:    4017852018407011111 0x55afa93755d8->"file"
                    3729804957429652673 0x55afa936ecc8->"<<ROOT>>"
string_set[  59]:   16579916199511856337 0x55afa9377178->"getln"
string_set[  61]:   14849783028996024063 0x55afa9377de8->"not"
string_set[  62]:    3182526723088623345 0x55afa9377778->"number"
string_set[  63]:    9611428142580219387 0x55afa93750b8->"fail"
string_set[  64]:   12597418120160561823 0x55afa9375758->"int"
string_set[  70]:    9201800968376702500 0x55afa9376b78->"getchr"
string_set[  76]:   15430489877420048373 0x55afa9378588->"<="
string_set[  78]:    4706206694200543916 0x55afa93750f8->","
load_factor = 0.468
bucket_count = 79
max_bucket_size = 3
