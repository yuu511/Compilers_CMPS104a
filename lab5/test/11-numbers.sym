main (7.5.1) {7} int function 
   number (7.6.4) {8} int variable local lval 0
   done (7.7.4) {8} int variable local lval 1

getln (6.30.1) {6} string function 

putchr (6.24.1) {1} void function 

getstr (6.29.1) {5} string function 

fail (6.22.1) {0} void function 

putint (6.25.1) {2} void function 

putstr (6.26.1) {3} void function 

getchr (6.28.1) {4} int function 

