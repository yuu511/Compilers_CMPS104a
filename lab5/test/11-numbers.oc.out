CONDITIONAL WHILE STATEMENT SYMBOL TOK_NOT 
CONDITIONAL IF STATEMENT SYMBOL TOK_LE 
